sandy/core/cov.py

```diff
@@ -2327,12 +2363,35 @@ def sample_distribution(dim, nsmp, seed=None, pdf='normal'):
     >>> sandy.cov.sample_distribution(2, 3, seed=11)
     array([[ 1.74945474, -0.286073  , -0.48456513],
            [-2.65331856, -0.00828463, -0.31963136]])
+
+    >>> sandy.cov.sample_distribution(2, 3, seed=11, pdf='lognormal')

```

Comment:
sample_distribution should be added to `__all__` and be called as `sandy.sample_distribution`